modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -20,6 +20,7 @@ The following table describes the CSI drivers that are installed with {product-t
 |AWS EBS | ✅ | - | ✅
 |AWS EFS | - | - | -
 |Google Cloud Platform (GCP) persistent disk (PD)| ✅ | - | ✅
+|Google Filestore | ✅ | ✅  | ✅
```

Comment:
Perhaps `Google Compute Platform`? We should probably Fix the line above as well.

Reply:
Done.